src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -120,10 +127,14 @@ export class FileService implements IFileService {
 		}
 
 		if (this.basePath && !this.options.disableWatcher) {
-			if (isWindows) {
-				this.setupWin32WorkspaceWatching();
+			if (this.options.useNsfwFileWatcher) {
```

Comment:
Not just yet, I haven't tested at all on Windows. I may do so after playing with it on Monday.